src/components/app/TabBar.css

```diff
@@ -18,10 +18,13 @@
 .tabBarTabWrapper {
   display: flex;
   min-width: 0; /* This makes the tab container actually shrinkable below min-content */
+  flex: 1;
   flex-flow: row nowrap;
   padding: 0;
   margin: 0 -0.5px; /* This combines with the -0.5px horizontal margin of .tabBarTab to clip off the first tab's 1px starting border */
   list-style: none;
+  overflow-x: auto;
```

Comment:
Oh, I was under the assumption that setting `overflow-x` to `auto` and leaving `overflow-y` unspecified made it so that `overflow-y` acted as if it was `hidden`. But that's actually not the case! It acts as if it was also set to `auto`. So we should actually specify both here, so that `overflow-y` is `hidden` and so that on platforms with overscroll rubberbanding you can't scroll the tab bar vertically. So it's better to write `overflow: auto hidden` here.

Reply:
Understood!